src/libpmempool/replica.c

```diff
@@ -75,10 +76,30 @@ check_flags_sync(unsigned flags)
 static int
 check_flags_transform(unsigned flags)
 {
-	flags &= ~(unsigned)PMEMPOOL_DRY_RUN;
+	flags &= ~PMEMPOOL_TRANSFORM_DRY_RUN;
 	return flags > 0;
 }
 
+/*
+ * replica_align_offset_length -- align data offset and length
+ *                                for the given part
+ */
+void
+replica_align_offset_length(size_t *offset, size_t *length,
```

Comment:
hmm, the only occurrence of this function that I could find is for bad blocks. Please mention this in the comment or in the function name - if someone were to use this for aligning off/len of the part, they it would be wrong (because it aligns ups the length - which might cause parts to overlap, same for offset).

Reply:
Changed to replica_align_badblock_offset_length